docs/_includes/deeplink.html

```diff
@@ -1,4 +1,2 @@
-<script src="{{ "/js/anchor.min.js" | relative_url }}"></script>
-<script>
-  anchors.add('h1, h2, h3, h4, h5, h6');
-</script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/anchor-js/3.2.2/anchor.min.js"></script>
```

Comment:
I think he means whether to use a cdn or just put the js file into assets for the page.